application/ui/src/features/inspect/stream/stream.tsx

```diff
@@ -82,37 +88,25 @@ const useStreamToVideo = () => {
     return videoRef;
 };
 
-export const Stream = ({
-    size,
-    setSize,
-}: {
-    size: { width: number; height: number };
-    setSize: Dispatch<SetStateAction<{ width: number; height: number }>>;
-}) => {
+export const Stream = ({ size, setSize }: StreamProps) => {
     const videoRef = useStreamToVideo();
-
     useSetTargetSizeBasedOnVideo(setSize, videoRef);
 
-    const { status } = useWebRTCConnection();
-
     return (
         <ZoomTransform target={size}>
-            <div style={{ gridArea: 'innercanvas' }}>
-                {status === 'connected' && (
-                    // eslint-disable-next-line jsx-a11y/media-has-caption
-                    <video
-                        ref={videoRef}
-                        autoPlay
-                        playsInline
-                        width={size.width}
-                        height={size.height}
-                        controls={false}
-                        style={{
-                            background: 'var(--spectrum-global-color-gray-200)',
-                        }}
-                    />
-                )}
-            </div>
+            <video
+                ref={videoRef}
+                muted
```

Comment:
Modern browsers don’t allow videos to autoplay unless the user has interacted with the page at least once. Muting the video lets us bypass that restriction.